src/dashboards/Chrono/components/TileRows/index.tsx

```diff
@@ -27,7 +27,7 @@ export function TileRows({
                 const icon = getIcon(data.type, iconColorType, data.subType)
                 const subLabel = createTileSubLabel(data)
                 return (
-                    <TableRow key={data.id}>
+                    <TableRow key={data.id} className="tilerows">
```

Comment:
Eg ser at i BusStop så heiter har denne eintallsform, "tilerow". Det gir kanskje meir meining, sidan dette er eitt element i ei liste? Må i så fall oppdatere i css-fila òg.

Reply:
Jeg er enig i at det ville vært bedre, men sånn applikasjonen er lagt opp for Chrono-viewet nå (hvis jeg har forstått rett) så er `TileRows` brukt for kollektivstopp og `TileRow` brukt for bysykkelposter, så `tilerow` er da allerede opptatt:/